src/Microsoft.AspNet.OData/Microsoft.AspNet.OData.csproj

```diff
@@ -42,6 +42,11 @@
       <HintPath>..\..\sln\packages\Microsoft.Spatial.7.12.2\lib\net45\Microsoft.Spatial.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL"/>
```

Comment:
There was an issue recently where customers reported an issue with WebAPI release 7.5.16. It was related to Newtonoft.Json Version 13.0.0.0 not being found. Here's the issue: https://github.com/OData/WebApi/issues/2701

Kindly verify whether that issue would affect this PR.

Reply:
so the issue was fixed in 7.5.17. Let me rebase and see what happens

Reply:
No it won't. There is a fix that was made that makes this version of Newtonsoft visible.